### ISSUE TITLE
Error: Unknown Amazon Bedrock model: meta.llama3-70b-instruct-v1:0

### ISSUE THREAD

**Issue by underyx** [promptfoo/promptfoo#884]
Thanks for the tool! Just tried to compare a bunch of providers via bedrock, and I'm getting a couple errors, e.g. for this provider config:

```yaml
  - id: bedrock:meta.llama3-70b-instruct-v1:0
    config:
      region: "us-west-2"
```

```
Error: Unknown Amazon Bedrock model: meta.llama3-70b-instruct-v1:0
    at getHandlerForModel (/usr/local/lib/node_modules/promptfoo/dist/src/providers/bedrock.js:95:11)
    at AwsBedrockCompletionProvider.callApi (/usr/local/lib/node_modules/promptfoo/dist/src/providers/bedrock.js:142:21)
    at Evaluator.runEval (/usr/local/lib/node_modules/promptfoo/dist/src/evaluator.js:273:62)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async processEvalStep (/usr/local/lib/node_modules/promptfoo/dist/src/evaluator.js:601:25)
```

Same issue for `mistral.mixtral-8x7b-instruct-v0:1`.

**Comment by typpo** [comment]
Thanks for flagging!  Support for llama, mistral, and cohere via Bedrock is added in #885

**Comment by underyx** [comment]
Amazing! Excited to try.

**Comment by typpo** [comment]
This is landed in 0.63.0, although I was not able to test all the bedrock providers (I don't have access to all).  Let me know if you run into any issues!

**Comment by underyx** [comment]
Yeah, exciting! Unfortunately none of them seem to _just work_. I created #912 #913 #914.

For the record, I think all requests for access to models are instantly auto-approved on AWS.
